indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -168,6 +168,16 @@
         has-variables (or (:has-variables variable-docs) (:has-variables service-docs))]
     (merge variable-docs service-docs {:has-variables has-variables})))
 
+(defn- get-granule-data-format
+  "Returns the Format field from
+  ArchiveAndDistributionInformation -> FileDistributionInformation"
+  [file-distribution-information]
+  (->> file-distribution-information
+       (map #(:Format %))
+       distinct
+       (remove #(nil? %))
+       concat))
```

Comment:
What is concat doing here?

Reply:
I saw a lot of the other indexed values using `mapcat` so I followed suit. I can also get rid of it very easily